scripts/run-draft-pr-scripts.ts

```diff
@@ -196,7 +202,20 @@ class DraftPRScriptRunner {
     const fullPath = join(this.projectRoot, scriptPath);
     const startTime = Date.now();
     
-    console.log(`🚀 Running script: ${scriptPath}`);
+    console.log(`🚀 ${this.isPreviewMode ? 'Would run' : 'Running'} script: ${scriptPath}`);
+    
+    if (this.isPreviewMode) {
+      const command = this.getExecutionCommand(scriptPath);
+      const duration = Date.now() - startTime;
+      console.log(`✅ Preview: Would execute command: ${command}`);
+      
+      return {
```

Comment:
Instead of early returning, pass `--preview` into the `executionCommand` returned